sql/mob_pools.sql

```diff
@@ -258,7 +258,7 @@ INSERT INTO `mob_pools` VALUES (202,'Apollyon_Scavenger','Apollyon_Scavenger',58
 INSERT INTO `mob_pools` VALUES (203,'Apollyon_Sweeper','Apollyon_Sweeper',84,0x0000300100000000000000000000000000000000,1,1,12,240,100,0,1,1,1,18,2048,0,4680,1159,4,0,0,0,256,85,84);
 INSERT INTO `mob_pools` VALUES (204,'Apsaras','Apsaras',197,0x00005C0100000000000000000000000000000000,1,1,7,240,100,0,1,0,0,4,0,0,0,3,0,0,0,0,0,197,197);
 INSERT INTO `mob_pools` VALUES (205,'Aquarian_Caster','Aquarian_Caster',153,0x01000D0512101420143014401450E36027701B80,1,1,0,0,100,0,0,0,0,0,0,0,0,1155,0,0,0,0,0,153,153);
-INSERT INTO `mob_pools` VALUES (206,'Aquarius','Aquarius',77,0x0000640100000000000000000000000000000000,7,7,4,180,100,0,1,0,0,2,257,0,0,1165,0,0,0,0,0,77,77);
+INSERT INTO `mob_pools` VALUES (206,'Aquarius','Aquarius',77,0x0000640100000000000000000000000000000000,7,7,4,180,100,0,1,0,0,2,6401,0,0,1165,0,0,0,0,0,77,77);
```

Comment:
What all is this guy supposed to be immune to?

Reply:
Is 257 (poison + sleep) sufficient or do I need to account for dark sleep and light sleep both separately?

Reply:
1 is all forms of sleep! So poison + sleep is good 😊